### PR TITLE
Remove the picture that doesn't work

### PR DESCRIPTION
**Pull request by JoGetBlock**

### CODE REVIEW

File: docs/05-resources/00-rpc-providers.md [hunk 1]
```diff
@@ -116,7 +116,6 @@ After going through the documentation, you can reserve an account in their [port
 
 After setting up an account and proceed to choose a plan.
 
-![](https://imgur.com/iqO3rE7)
```
Comment (MarkSackerberg) [inline docs/05-resources/00-rpc-providers.md:119]: using the following url instead should work.
```
![](https://imgur.com/iqO3rE7.png#radius)
```

File: docs/05-resources/00-rpc-providers.md [hunk 2]
```diff
@@ -116,7 +116,6 @@ After going through the documentation, you can reserve an account in their [port
 
 After setting up an account and proceed to choose a plan.
```
Comment (MarkSackerberg) [inline docs/05-resources/00-rpc-providers.md:117]: > After setting up an account and proceed to choose a plan.

The "and" in the middle of the sentence does not really make sense, does it?

**Comment by MarkSackerberg** [comment]
Thanks for the proposal!

It's absolutely possible to make the image appear though, it does not have to be removed 👍

**Comment by JoGetBlock** [comment]
You're right

 After setting up an account, proceed to choose a convenient plan.

**Review by lorisleiva — APPROVED** [review]
Thanks @MarkSackerberg for the review.

**Pull request merged**